src/cryptoadvance/specter/templates/includes/tx-table.html

```diff
@@ -175,6 +175,7 @@
         }
         .table-container {
             width:  100%;
+            height: 100vh !important;
```

Comment:
@k9ert I think so cause I have tested :sweat_smile:

Reply:
it works, but the reason `!important` is required is because it conflicts with a css value that we already set (https://github.com/cryptoadvance/specter-desktop/blob/69eb530b73700a448cbc81860abe37afd7800ca9/src/cryptoadvance/specter/templates/includes/tx-table.html#L662
)

Reply:
> it works, but the reason `!important` is required is because it conflicts with a css value that we already set (
> 
> https://github.com/cryptoadvance/specter-desktop/blob/69eb530b73700a448cbc81860abe37afd7800ca9/src/cryptoadvance/specter/templates/includes/tx-table.html#L662
> 
> 
> )

Yeah exactly, this forced me to use `!important`

Reply:
I think a cleaner fix is to remove the conflicting code so that you dont need to use `!important`